src/services/render-widget-service.tsx

```diff
@@ -8,12 +8,18 @@ import { EditorMetadata } from "../editor/editor-metadata";
 import { WidgetModel } from "../editor/interfaces";
 import { RenderContext } from "./render-context";
 import { RequestContext } from "./request-context";
+import { GridComponent } from "../components/grid/grid-component";
+import { ButtonComponent } from "../components/kendo-button/kendo-button";
+import { TextBoxComponent } from "../components/kendo-textbox/kendo-textbox";
 
 export const TYPES_MAP = {
     "SitefinityContentBlock": ContentBlock,
     "SitefinitySection": Section,
     "SitefinityContentList": ContentList,
-    "SitefinityChart": ChartComponent
+    // "SitefinityChart": ChartComponent,
```

Comment:
commented out but the code is there, or am I missing something?

Reply:
Chart is deliberately removed since it does not support server side rendering